gitbook_client.ts

```diff
@@ -1,174 +1,112 @@
-interface GitbookClientOptions {
-  version?: "v1";
-  spaceId: string;
-  gitbookUrl: string;
+import { GitbookClientOptions, GitbookContent, GitbookPage, GitbookSearchNode, GitbookSpace, PathWeight } from './types/types.ts'
```

Comment:
`./types/types.ts` looks odd.

Maybe create an `index.ts` or `index.d.ts` under `types` directory, and simplify the import